docs/asyncio_categorizing_functions.md

```diff
@@ -62,6 +62,27 @@ These are functions that are safe to run both in a thread and inside the event l
 
 There is no special annotation to mark functions as part of this category and care should be taken when using these functions from inside the event loop. When in doubt, look at their implementation.
 
+## Calling functions in async context
+
+There are mutliple ways to execute the different types of functions when being in the event loop (async context). All of the listed functions are methods of the HomeAssistant object.
+
+- Call sync functions: `async_add_executor_job`
+
+   They will be scheduled in the executor thread pool and can be awaited.
+- Call coroutines: `async_create_task`
```

Comment:
Run coroutines in a parallel process. If you normally want to run coroutines, just await them. This is like starting a function in a thread if you are in sync context. 

FYI Any `async def` function is a coroutine function that produces a coroutine when called.